atoMEC/mathtools.py

```diff
@@ -194,6 +239,46 @@ def fd_int_complete(mu, beta, n):
     return I_n
 
 
+def ideal_entropy_int(mu, beta, n):
+    r"""
+    Compute the entropy for the ideal electron gas (without prefactor) - see notes.
+
+    Parameters
+    ----------
+    mu : float
+        chemical potential
+    beta: float
+        inverse temperature
+    n : int
+        order of Fermi-Dirac integral (see notes)
+
+    Returns
+    -------
+    I_n : float
+        the complete fermi-dirac integral
+
+    Notes
+    -----
+    The entropy of an ideal electron gas is defined as
+
+    .. math::
+        I_n(\mu,\beta) = \int \mathrm{d}\epsilon \epsilon^{n/2}
```

Comment:
The integral limits are fix, right?
```suggestion
        I_n(\mu,\beta) = \int_0^\infty \mathrm{d}\epsilon\ \epsilon^{n/2}
```